packages/tfchain_client/src/client.ts

```diff
@@ -351,15 +365,19 @@ class Client extends QueryClient {
           resultSections.length > 0
         )
           section = resultSections[0];
+        //TODO no need to do the following lines only check if the error is instanceof base error and edit the message
```

Comment:
Is this to be worked on in another pr? if so please open an issue and mention it here

Reply:
removed